src/cli/backup/exchange.go

```diff
@@ -273,20 +273,23 @@ func createExchangeCmd(cmd *cobra.Command, args []string) error {
 
 	sel := exchangeBackupCreateSelectors(user, exchangeData)
 
-	users, err := m365.UserPNs(ctx, acct)
+	// TODO: log/print recoverable errors
+	errs := fault.New(false)
+
+	users, err := m365.UserPNs(ctx, acct, errs)
```

Comment:
along the same lines as other comments but flagging here too. The CLI should probably check `errs.Errs()` because if we don't we may change the behavior of corso when we don't mean to